devdoc/userdoc/README.md

```diff
@@ -0,0 +1,27 @@
+# User Documentation
+
+This directory contains the documentation specifically targeted to users of the HPCC system.  The documentation availalbe here is less formal and more ad-hoc than the published HPCC documentation (see https://github.com/hpcc-systems/HPCC-Platform/tree/master/docs if you would like to contribute to our official docs)
```

Comment:
Readability:  "This directory contains the documentation specifically targeting users of the HPCC system."
Typo:  "available"
Formatting: "HPCC documentation (see https://github.com/hpcc-systems/HPCC-Platform/tree/master/docs if you would like to contribute to our official docs)" -> "[HPCC documentation](see https://github.com/hpcc-systems/HPCC-Platform/tree/master/docs if you would like to contribute to our official docs)"